doc/releases/release-notes-2.6.rst

```diff
@@ -181,10 +181,20 @@ Architectures
 
     * Added support for null pointer dereferencing detection in Cortex-M.
 
-    * Added initial support for Arm v8.1-m and Cortex-M55
+    * Added initial support for Arm v8.1-m and Cortex-M55.
 
     * Added support for preempting threads while they are performing secure calls in Cortex-M.
 
+    * Added support for memory region generation by the linker based on device tree
+      node information in Cortex-M.
+
+    * Cleaned up definitions of SoC-specific memory regions in the common Cortex-M linker script.
+
+    * Added support for clearing NXP MPU region configuraton during Zephyr early boot stage.
```

Comment:
"configuration"

Reply:
thanks.
and btw, if I missed any key changes for Cortex-R, let me know.